packages/inventory-insights/src/Constants.js

```diff
@@ -1,24 +1,19 @@
+/* eslint-disable camelcase */
 export const BASE_FETCH_URL = '/api/insights/v1/';
 
-export const FILTER_CATEGORIES = [
-    {
-        type: 'checkbox', title: 'Ansible support', urlParam: 'has_playbook', values: [
-            { label: 'Has support', value: true },
-            { label: 'No support', value: false }
+export const FILTER_CATEGORIES = {
+    has_playbook: {
+        type: 'radio', title: 'Ansible support', urlParam: 'has_playbook', values: [
+            { label: 'Ansible remediation support', value: 'true' },
```

Comment:
Is it intentional to have these `value`s boolean string?

Reply:
yeah, conditional filter will eat up the false and or undefined (resort to displaying the array position of the entry) so we do strings to avoid that

Reply:
Oh I get it, might want to open an issue to address this.

Reply:
https://github.com/RedHatInsights/frontend-components/issues/250